app/views/schedules/_root_task_fields.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="nested-fields form-inline">
+  <div class="form-group">
+    <div class="input-group">
+        <% if f.object.id.nil? %>
```

Comment:
Since this partial is only meant to be used when we're creating the Schedule, the Task's id will always be nil. You can remove the other case.